### PR TITLE
Create option `--new-chain` to explicitly request a chain from a faucet

### PR DESCRIPTION
**Pull request by ma2bd**
## Motivation

`linera wallet init --faucet URI` creates a (default) chain implicitly. This caused some confusion because `linera wallet init --genesis FILE` does not.

## Proposal

* Require that `--new-chain` is passed `linera wallet init --faucet URI --new-chain`
* Create type `FaucetOption`

## Test Plan

CI

## Release Plan

- Need to bump the major/minor version number in the next release of the crates.
- Need to update the developer manual.

### CODE REVIEW

File: linera-service/src/cli_wrappers/local_kubernetes_net.rs [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 // SPDX-License-Identifier: Apache-2.0
 
-use super::{
-    docker::DockerImage, kind::KindCluster, kubectl::KubectlInstance, util::get_github_root,
-};
 use crate::{
-    cli_wrappers::{helmfile::HelmFile, ClientWrapper, LineraNet, LineraNetConfig, Network},
+    cli_wrappers::{
+        docker::DockerImage, helmfile::HelmFile, kind::KindCluster, kubectl::KubectlInstance,
+        util::get_github_root, wallet::FaucetOption, ClientWrapper, LineraNet, LineraNetConfig,
```
Comment (andresilva91) [inline linera-service/src/cli_wrappers/local_kubernetes_net.rs:7]: I think `wallet::FaucetOption` needs to be under `#[cfg(any(test, feature = "test"))]`

Comment (ma2bd) [reply linera-service/src/cli_wrappers/local_kubernetes_net.rs]: right!

File: linera-service/src/cli_wrappers/local_kubernetes_net.rs [hunk 2]
```diff
@@ -20,8 +20,12 @@ use linera_base::data_types::Amount;
 use std::{path::PathBuf, sync::Arc};
 use tempfile::{tempdir, TempDir};
 use tokio::process::Command;
+
 #[cfg(any(test, feature = "test"))]
-use {crate::util::current_binary_parent, tokio::sync::OnceCell};
+use {
+    crate::cli_wrappers::wallet::FaucetOption, crate::util::current_binary_parent,
```
Comment (andresilva91) [inline linera-service/src/cli_wrappers/local_kubernetes_net.rs:26]: nit
```suggestion
    crate::{util::current_binary_parent, cli_wrappers::wallet::FaucetOption},
```

**Review by andresilva91 — APPROVED** [review]
Nice! Thank you! One small nit, not blocking
[shipit](https://app.graphite.dev/macros/linera-io)

**Pull request merged**